openstacksdk_plugin/constants.py

```diff
@@ -104,13 +106,17 @@
     'provisioned {2}: {3}, quota: {4}'
 
 # General constants
+OPENSTACK_RESOURCE_UUID = 'uuid'
+OPENSTACK_PORT_ID = 'port_id'
+OPENSTACK_NETWORK_ID = 'net_id'
 PS_OPEN = '<powershell>'
 PS_CLOSE = '</powershell>'
 INFINITE_RESOURCE_QUOTA = -1
 SERVER_ACTION_STATUS_DONE = 'DONE'
 SERVER_ACTION_STATUS_PENDING = 'PENDING'
 SERVER_REBUILD_STATUS = 'rebuild_done'
 SERVER_REBUILD_SPAWNING_STATUS = 'rebuild_spawning'
+SERVER_ADMIN_PASSWORD = 'password'
 IDENTITY_USERS = 'users'
 IDENTITY_ROLES = 'roles'
 IDENTITY_QUOTA = 'quota'
```

Comment:
Did you see [this](http://mauveweb.co.uk/posts/2014/12/dont-name-strings.html)?

Reply:
will check it now